From: uazo <uazo@users.noreply.github.com>
Date: Wed, 29 May 2024 13:49:01 +0000
Subject: Enable Do-Not-Track by default

Enables the "Do Not Track" Signal (https://wikipedia.org/wiki/Do_Not_Track) by default to opt-out of tracking from websites & mitigate fingerprinting.

Initial Commit: https://github.com/uazo/cromite/commit/547f3a53dbb8c203062719a34ae89c7a58a146d6
Category: Privacy
License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 components/privacy_sandbox/tracking_protection_prefs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/privacy_sandbox/tracking_protection_prefs.cc b/components/privacy_sandbox/tracking_protection_prefs.cc
--- a/components/privacy_sandbox/tracking_protection_prefs.cc
+++ b/components/privacy_sandbox/tracking_protection_prefs.cc
@@ -79,7 +79,7 @@ void RegisterProfilePrefs(PrefRegistrySimple* registry) {
       prefs::kFingerprintingProtectionEnabled, false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   registry->RegisterBooleanPref(
-      prefs::kEnableDoNotTrack, false,
+      prefs::kEnableDoNotTrack, true,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   registry->RegisterBooleanPref(prefs::kEnableGPC, true);
 }
--
